### DISCUSSION TITLE
HTML)How can I combine the heights of the tables equally?

### DISCUSSION THREAD

**Discussion by dohun9196** [community/community#21479] [Other Feature Feedback, Questions, & Ideas]
<p>i want row b1,b2 to have same  height</p>
<table>
        <thead>
          <tr>
            <th>head</th>
          </tr>
</thead>
           <tbody>
             <tr>
               <td><img></td>
               <td>@@</td>
              <td>$$</td>
            </tr>
           <tr>
             <td>a1</td>
             <td>b1</td>
          </tr>
          <tr>
             <td>a2</td>
          </tr>
          <tr>
             <td>a3</td>
             <td>b2</td>
        </tr>
 </tbody>
</table>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/stivehum">@StiveHum</a>, welcome to the GitHub Support Community!</p>
<p>You can adjust the height of all table cells in a row with the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style" rel="nofollow noopener">style attribute</a>. Like this for example:</p>
<pre><code class="lang-auto">&lt;td style="height:100px;"&gt;table cell&lt;/td&gt;
</code></pre>
<p>Tables are designed to be flexible, however, so this will only set a minimum height.</p>
